packages/notion-client/src/notion-api.ts

```diff
@@ -235,7 +235,7 @@ export class NotionAPI {
         // console.log(block, source)
 
         if (source) {
-          if (source.indexOf('youtube') >= 0 || source.indexOf('vimeo') >= 0) {
+          if (!source.includes('secure.notion-static.com')) {
```

Comment:
I always only encountered urls with `secure.notion-static.com` that needed to be signed, there may be more endpoints than this one